pkg/apis/cluster/v1alpha1/cluster_consts.go

```diff
@@ -21,19 +21,22 @@ import (
 )
 
 const (
-	// ReadyCondition defines the Ready condition type that summarizes the operational state of a Cluster.
-	ReadyCondition capiv1.ConditionType = "Ready"
-
-	// InfrastructureProviderProvisionedCondition reports on wheter the infra resource is provisioned.
+	// InfrastructureProviderProvisionedCondition reports on wheter the infrastructure resource is provisioned.
 	InfrastructureProviderProvisionedCondition capiv1.ConditionType = "InfrastructureProviderProvisioned"
-	// InfrastructureProviderProvisionFailedReason (Severity=Error) documents that the infra provisioning failed.
+	// InfrastructureProviderProvisionFailedReason (Severity=Error) documents that the infrastructure provisioning failed.
 	InfrastructureProviderProvisionFailedReason = "InfrastructureProviderProvisionFailed"
+	// InfrastructureProviderProvisionReadyReason (Severity=Info) documents that the infrastructure is not ready.
```

Comment:
is it ready or not ready? not ready does not consistent with `InfrastructureProviderProvisionReadyReason`